com.unity.netcode.gameobjects/Tests/Runtime/NetworkObject/NetworkObjectOwnershipTests.cs

```diff
@@ -147,58 +145,61 @@ public IEnumerator TestOwnershipCallbacks([Values] OwnershipChecks ownershipChec
                 m_ServerNetworkManager.SpawnManager.RemoveOwnership(m_OwnershipNetworkObject);
             }
 
-            var serverObject = m_ServerNetworkManager.SpawnManager.SpawnedObjects[ownershipNetworkObjectId];
-            var clientObject = m_ClientNetworkManagers[0].SpawnManager.SpawnedObjects[ownershipNetworkObjectId];
-            Assert.That(serverObject, Is.Not.Null);
+            var authorityObject = authority.SpawnManager.SpawnedObjects[ownershipNetworkObjectId];
```

Comment:
Really like your clean up within NetcodeIntegrationTest and how that improves the test flow... :godmode: